ambry-store/src/main/java/com/github/ambry/store/PersistentIndex.java

```diff
@@ -99,7 +99,7 @@ public int compare(IndexEntry e1, IndexEntry e2) {
   final HardDeleter hardDeleter;
   final Thread hardDeleteThread;
 
-  private final Log log;
+  protected final Log log;
```

Comment:
May I ask why this is `protected`?

Reply:
Because previous in BlobStoreCompactorTest I need to call state.index.log. But since you suggest for clean up journal we can directly clear the Hashmap, so there's no need to bring in the active log segment name. So after I tested, I will change it back.